popup2.js

```diff
@@ -1,8 +1,48 @@
-window.addEventListener('load',function(){
-	document.getElementById('submitbutton').addEventListener("click", function() {
+window.addEventListener('load',function() {
+	document.getElementById('submitbutton').addEventListener('click', function() {
```

Comment:
nit: space between function() and {
nit: space after commas
nit: stick to either single or double quotes